src/plugins/session-timeout/session-timeout.js

```diff
@@ -36,7 +36,13 @@ var $modal, $modalLink, countdownInterval, i18n, i18nText,
 		refreshCallbackUrl: null,	/* refresh callback if using AJAX keepalive (no default) */
 		logouturl: "./",			/* logout URL once the session has expired */
 		refreshOnClick: true,		/* refresh session if user clicks on the page */
-		refreshLimit: 200000		/* default period of 2 minutes (ajax calls happen only once during this period) */
+		refreshLimit: 200000,		/* default period of 2 minutes (ajax calls happen only once during this period) */
+		method: "POST",				/* the request method to use */
+		additionalData: null,		/* additional data to send with the request */
+		sessionaliveCallback:		/* callback function used to check the server response */
+		function( response ) {
```

Comment:
Can you make this be on the previous line and indent the content of the function

```
sessionaliveCallback: function( response ) {		/* callback function used to check the server response */
	return response.replace( /\s/g, "" ) === "true";
}
+		
```